resources/views/tiny-editor.blade.php

```diff
@@ -22,8 +22,8 @@ class="relative z-0"
                     tinymce.init({
                         target: $refs.tinymce,
                         language: '{{ $getInterfaceLanguage() }}',
-                        skin: typeof theme != 'undefined' ? theme : 'light',
```

Comment:
This line prevents an error established when **darkMode** is disabled. revert it